README.md

```diff
@@ -954,42 +954,72 @@ Used for EV chargers.
-in   | cmd.smart_charge.get_report    | null       |                         | 
-out  | evt.smart_charge.report        | bool       |                         |
+in   | cmd.smart_charge.set            | bool     |                         | true = enable smart charging
+in   | cmd.smart_charge.get_report     | null     |                         | 
+out  | evt.smart_charge.report         | bool     |                         |
 -|||
-in   | cmd.rated_current.get_report   | null       |                         | Get the rated or max current for ths CP
-out  | evt.rated_current.report       | int        |                         |
+in   | cmd.rated_current.get_report    | null     |                         | Get the rated or max current for ths CP
+out  | evt.rated_current.report        | int      |                         |
 -|||
-in   | cmd.current_session.get_report | null       |                         | Command for getting energy (kWh) for current session.
-out  | evt.current_session.report     | float      |                         |
+in   | cmd.current_session.get_report  | null     |                         | Command for getting energy (kWh) for current session.
+out  | evt.current_session.report      | float    |                         |
 -|||
-out  | evt.error.report               | string     |                         | 
+in   | cmd.electric_current.set        | object   |                         | Command for setting an electric current.
+in   | cmd.electric_current.get_report | null     |                         |
+out  | evt.electric_current.report     | object   |                         |
+-|||
+out  | evt.error.report                | string   |                         | 
 
 #### Interface props
 
-Name          | Value example                                                 | Description
---------------|---------------------------------------------------------------|-------------
-`sup_states`  | ["disconnected", “requesting”, “charging”, “ready\_to\_charge”, "requesting", "finished", "reserved", "unavailable", "error", "unknown"]  | State of the CP
-`sup_access`  | ["rfid", "open", "app", "local"]                              | Access level for using CP
-`sup_errors`  | []                                                            | Error from CP
+Name          | Value example                                                                                                                            | Description
+--------------|------------------------------------------------------------------------------------------------------------------------------------------|-------------
+`sup_states`  | ["disconnected", “requesting”, “charging”, “ready\_to\_charge”, "requesting", "finished", "reserved", "unavailable", "error", "unknown"] | State of the CP
+`sup_access`  | ["rfid", "open", "app", "local"]                                                                                                         | Access level for using CP
```

Comment:
Same with sup_access. If it was not used or it is not interpreted in any way by UI, let's remove it from spec. Double check with @GenixPL and Markus.

Reply:
Markus confirmed it's not used anywhere inside edge apps. Waiting for confirmation from Łukasz...

Reply:
Confirmed from Łukasz's side, too. Removing it...

Reply:
Fixed.